.github/scripts/verify_artifact.sh

```diff
@@ -70,11 +70,11 @@ function verify_rpm {
   case "${artifact_path}" in
     *.i386.rpm)
       docker_platform="linux/386"
-      docker_image="i386/centos:7"
```

Comment:
There's a huge deprecation notice for this repo [here](https://hub.docker.com/r/i386/centos/), and centos:9 has moved to quay.io and uses "streaming" images now. Rather than make that big of a chance, I simply switched to debian which supports the same `linux/386` architecture.